sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -188,6 +188,7 @@ def online_read(
         table: FeatureView,
         entity_keys: List[EntityKeyProto],
         requested_features: Optional[List[str]] = None,
+        batch_size: int = 20,
```

Comment:
why do we even want to let caller pass this parameter? why not always use batch_size of 40?

Reply:
@vlin-lgtm setting `batch_size` at 40 might be a good idea to minimize overall latency but I'm more inclined to give that control to the user to specify the number of items to send in a `batch_get_item`.

Reply:
@vlin-lgtm do you with how many `ReadCapacityUnits` is the DynamoDB table configured when created?

Reply:
This does seem weird since it no longer follows the online store contract. 

Would it be sufficient to add this batch size to the DynamoDBOnlineStoreConfig (aka via feature_store.yaml)?

Reply:
> @vlin-lgtm do you with how many ReadCapacityUnits is the DynamoDB table configured when created?

I believe tables are created with "on-demand" RCU and WCU rather than with provisioned with some initial units. This can be changed by users at a later time.

Reply:
> I'm more inclined to give that control to the user to specify the number of items to send in a batch_get_item.

What are the trade-offs? If the worry is RCU, then sending this in different batch sizes hardly changes anything, it will most likely consume the same RCU. In fact, even without this PR, it would probably consume the same RCU today.

Reply:
FWIW, AWS' DynamoDB Java SDK hardcode a batch_size of 100 [here](https://github.com/willglynn/aws-sdk-for-java/blob/master/src/main/java/com/amazonaws/services/dynamodb/datamodeling/DynamoDBMapper.java#L878). Also, as you pointed out in [AWS' doc](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchGetItem.html) that 
> If you request more than 100 items, BatchGetItem returns a ValidationException with the message "Too many items requested for the BatchGetItem call."

So I think 40 makes sense because DDB's record size limit is 400KB.

Reply:
@vlin-lgtm ok, will set the default `batch_size` to 40 without the user specifying this value.

Reply:
> Would it be sufficient to add this batch size to the DynamoDBOnlineStoreConfig (aka via feature_store.yaml)?

do you mean setting `batch_size` as a general configuration to interact with DynamoDB? Can the user modify these values?

Reply:
Depends on what you mean by user. But yeah anybody who is setting up feast and calling get_online_features would have their own `feature_store.yaml`. Then you'd have a yaml file like

```yaml
project: my_project
registry: data/registry.db
provider: aws
online_store:
  type: dynamodb
  region: %AWS_REGION%
  batch_size: 40
```

which would populate the `DynamoDBOnlineStoreConfig` in this class